### ISSUE TITLE
Are you likely to get banned by using Chiaki?

### ISSUE THREAD

**Issue by joshbarrass** [thestr4ng3r/chiaki#171]
Just a little paranoid about this. Is there any evidence that Sony are banning people for using Chiaki? Are there any known cases where it has happened? Does Chiaki actually violate the TOS?

**Comment by thestr4ng3r** [comment]
Of course I cannot give any guarantees and I have not read their TOS, so consider that for the statement below.

tl;dr don't worry, it's safe.

Long Version:
Chiaki is essentially just a clone of their original RemotePlay client and behaves in almost the exact same way, so even detecting the use of it would be non-trivial for the console and would have to rely on subtle differences that could be easily fixed.
At the same time, I don't really see a reason why Sony would ban a user of Chiaki at all since, in contrast to jailbreaking for example, it does not enable anything such as cheating or piracy, which are the main reasons why Sony would have anything against it in the first place.
In practice, I am not aware of anybody getting banned and I have never received any negative feedback from Sony. While reversing the protocol, I bombarded my PS4 over months with partially broken packets which even made the RemotePlay server on the console crash many times and send a crash report to Sony, all while being logged in with my personal PSN account and it's still not banned.

**Comment by joshbarrass** [comment]
That puts me more at ease. I guess tolerating Chiaki means they don't have to make their own clients for Linux, either.
